go.mod

```
@@ -137,7 +138,10 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
-require github.com/google/go-tdx-guest v0.2.2 // indirect
+require (
+	github.com/google/go-attestation v0.5.0 // indirect
+	github.com/google/go-tdx-guest v0.2.2 // indirect
+)
 
 require (
 	cloud.google.com/go v0.110.2 // indirect
```

```suggestion
require (
	github.com/google/go-attestation v0.5.0 // indirect
	github.com/google/go-tdx-guest v0.2.2 // indirect
	cloud.google.com/go v0.110.2 // indirect
```
nit: group dependencies